daemon/cmd/cells.go

```diff
@@ -78,6 +80,8 @@ var (
 
 		// Auth is responsible for authenticating a request if required by a policy.
 		auth.Cell,
+
+		cell.Provide(newPolicyMess),
```

Comment:
I approve this intermediate step. let's add a comment here what it's setting up.